prReview.js

```diff
@@ -0,0 +1,7 @@
+function findFolderByName(n) {
+	try {
+		return n === null ? null : (Folder.find({ name: n }) || Folder.create({ name: n }));
```

Comment:
We shouldn't be creating a new folder for every non-existing search within this function.
Following the [SOLID](https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design) principle, the "S" (Single Responsibility Principle) state that a class/function should have one and only one reason to change, meaning that a class/function should have only one job.